content/news/XXXX-XX-XX-mixxx-2-3-0-released.md

```diff
@@ -0,0 +1,138 @@
+title: Mixxx 2.3.0 released
+authors: Jan Holthuis
+tags: 2.3, 2.3.0, release announcement
+status: draft
+comments: yes
+
+We are proud to announce the release of Mixxx 2.3.0!
+If you want to give it a spin, you can now [download it]({filename}/pages/download.md#stable) on Windows, macOS and Linux.
```

Comment:
Can we remove the it from the link, that looks too much like a order to me.

Reply:
Then the download is a bit unobvious. How about:

```suggestion
If you want to give it a spin, [you can download it now]({filename}/pages/download.md#stable) on Windows, macOS and Linux.
```

Reply:
Yes, also nice.